src/entities/entity.cpp

```diff
@@ -54,6 +53,8 @@ void Entity::move(manager::Level *levelManager) {
                 break;
             case enums::CollisionType::ARTIFACT:
                 // TODO(simo)
+                levelManager->getPlayer()->life_+=6;
+                setPosition(Position(new_x, new_y));
```

Comment:
qui non dovrebbe anche eliminare l'oggetto?
perchè se non lo elimini o non gli dici di smettere di essere printato, non lo printa perchè il player ci passa sopra
ma se fa il refresh della finestra si rivede

Reply:
Forse devi ancora finirlo?

Reply:
Sì, dovrebbe essere eliminato, ma una cosa strana, tra l'altro, è che quando ci ripassi sopra, non aumenta più la vita, anche in se in teoria, dato che non l'ha eliminato, dovrebbe aumentare la vita ancora una volta.

Reply:
E' risolto, in level/level.cpp all'interno della funzione getCollision ho gestitio il tutto e ho eliminato l'artefatto dalla lista degli artefatti, si dovrebbe fare in un modo più carino ma per ora va